src/machine-learning/machine-learning.ts

```diff
@@ -187,11 +184,10 @@ export class MachineLearning implements FirebaseServiceInterface {
           throw new FirebaseMachineLearningError(
             'internal-error',
             `Error during signing upload url: ${err.message}`);
-
-        });
+        }) as Promise<ModelContent>;
     }
 
-    return Promise.resolve(modelContent);
+    return Promise.resolve(modelContent) as Promise<ModelContent>;
```

Comment:
If you type `modelContent` you won't need the cast here.

Reply:
As above. I need to cast at the end after all the assignments are complete.